### ISSUE TITLE
commission methods

### ISSUE THREAD

**Issue by wa1kb0y** [wildlink/wildlink-api-php#9]
Hi, found some issues:
1. `getAppCommissionsSince()` returns array of arrays, without nested array with `Commission` key, its ok, just need to fix docs.
2. i dont see difference when using since argument in `getAppCommissionsSince()`. Every time got same results (all data).
3. `getPagedCommissions()` returns an error `either cursor or start_modified_date parameter is required`. Maybe also need to add `since` argument for this method.

**Comment by CydeSwype** [comment]
Thanks for reporting these!  I'll tackle these today and tomorrow and follow up with the resolution details.

**Comment by CydeSwype** [comment]
1. Fixed the Readme and added the other looping method
1. This appears to be regression in the API, will follow up with our devs to get this fixed.
1. Still looking at this one... I don't think I intended for clients to call this method.  I intended for the CommissionList class to call this as a helper method, but I should reconsider how this method is organized or at least have it respond differently.  Will address this tomorrow.
